packages/conf-scdoc/conf-scdoc.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+authors: "lthms@soap.coffee"
+maintainer: "lthms@soap.coffee"
+homepage: "https://git.sr.ht/~sircmpwn/scdoc"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+build: ["scdoc" "-v"]
+depexts: [
+  ["scdoc"] {os-family = "arch" | os-family = "archlinux"}
```

Comment:
```suggestion
  ["scdoc"] {os-family = "arch"}
```
I think this is what needs to be used here but keeping both does no harm

Reply:
For what it is worth, I’ve borrowed this from another `conf-` package (though I don’t remember which one).